wp-parsely.php

```diff
@@ -721,7 +721,29 @@ public function insert_parsely_page() {
 
 		$current_url = $this->get_current_url();
 
-		if ( in_array( get_post_type(), $parsely_options['track_post_types'], true ) && 'publish' === $post->post_status ) {
+		if ( is_home() ) {
+			$parsely_page['@type'] = 'WebPage';
+			// If front page is set to display a static page, get the URL of the posts page.
+			if ( 'page' === get_option( 'show_on_front' ) ) {
+				$parsely_page['url'] = get_permalink( get_option( 'page_for_posts' ) );
+			} else {
+				$parsely_page['url'] = get_home_url();
+			}
+			$parsely_page['headline'] = $this->get_clean_parsely_page_value( get_bloginfo( 'name', 'raw' ) );
+		} elseif ( is_front_page() ) {
+			$parsely_page['headline'] = $this->get_clean_parsely_page_value( get_bloginfo( 'name', 'raw' ) );
+			$parsely_page['url']      = get_home_url();
+			$parsely_page['@type']    = 'WebPage';
+		} elseif ( is_archive() ) {
```

Comment:
I know we discussed re-ordering these. But now that I'm looking at it, I'm wondering if one of the reasons was to allow customers to override the defaults of whether we treat something as a page or a post. I guess I'm wondering if there are cases where `is_home`, `is_front_page`, or `is_archive` returns true AND it's also in the `track_post_types` array. :thinking: